web/layout_attributes/nesting_flex_layouts/my_element.html

```diff
@@ -1,26 +1,26 @@
 <link rel="import" href="packages/polymer/polymer.html">
-
-<polymer-element name="my-element" noscript layout vertical>
+<dom-module id="my-element" ><!-- layout vertical -->
+  <!--<style>-->
```

Comment:
Sorry for that, I forgot that successive commits get added to open PRs on the same branch. I committed this to be able to link to it in dart-lang/polymer-dart#584 
Thanks a lot for your comments! Really helpful to get up-to-speed with recent changes in Polymer.